packages/@dataform/assertion_utils/BUILD

```diff
@@ -0,0 +1,35 @@
+load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("//:version.bzl", "DF_VERSION")
+load("//packages:index.bzl", "pkg_json", "pkg_npm_tar")
+
+package(default_visibility = ["//visibility:public"])
+
+ts_library(
+    name = "assertion_utils",
+    srcs = glob(["**/*.ts"]),
+    deps = [
+        "@npm//@types/node",
+    ],
+)
+
+pkg_json(
+    name = "json",
+    package_name = "@dataform/assertion_utils",
+    description = "Assertion utilities for Dataform.",
+    external_deps = [],
+    layers = [
+        "//:package.json",
+        "//packages/@dataform:package.layer.json",
+    ],
+    main = "index.js",
+    types = "index.d.js",
```

Comment:
`index.d.ts` not `index.d.js`, or am i missing something? wait, i thought we couldn't generate types?

Reply:
Woops! Thanks.
This doesn't use rollup as it's a self contained package, so we can keep them here.

Reply:
(Once I can do .d.ts files, I'll make this all consistent)